src/test/java/ru/ayaz/CommentTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class CommentTest {
+
+    @Test
+    public void shouldBeEqualWhenReferencesAreSame() {
+        Comment comment = new Comment("Ayaz", "Nice video!");
+        Comment anotherComment = comment;
+
+        assertTrue(comment.equals(anotherComment));
+    }
+
+    @Test
+    public void shouldNotBeEqualWhenInstancesAreDifferent() {
```

Comment:
🍎 Issue
I would say that this test is redundant.
It's an anomaly detection test, but its kind of weird )
We don't need anomaly detection tests for "hashcode" or "equals" cause they are generated by IDEA. There shouldn't be any anomalies of such kind.